strikecircle/tasks.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+from .env import env
+from logging import getLogger
+from django.core.serializers import serialize
+from strikecircle.models import Pledge, StrikeCircle
+
+from airtable import Airtable
+from background_task import background
+
+
+logger = getLogger(__name__)
+
+
+API_KEY = env("AIRTABLE_API_KEY")
+BASE_KEY = env("AIRTABLE_BASE_KEY")
+
+
+# TODO: Deploy supervisord in prod to run tasks https://django-background-tasks.readthedocs.io/en/latest/#running-tasks
+
+
+@background(schedule=10, queue="airtable-export")
+def export_strike_circle_to_airtable(strike_circle_id):
```

Comment:
Seems like `export_strike_circle_to_airtable` and `export_pledge_to_airtable` could be abstracted into a single `export_model_to_airtable` function that takes a model and a table name in addition to an id? I'm definitely willing to back off this idea if you think it's a bad idea.